recipe/meta.yaml

```diff
@@ -31,6 +32,7 @@ requirements:
     - tk
   run:
     - python
+    - vs2015_runtime 14.00.23026.0 0   # [win and py35]
```

Comment:
Of course, see this [comment]( https://github.com/conda-forge/staged-recipes/issues/666#issuecomment-221125814 ).